packages/toolkit/src/useEnvironment.ts

```diff
@@ -90,74 +86,39 @@ function configureApm(
   }
 }
 
-function configureProvider(
```

Comment:
If we migrate to Aragon Connect as the data provider we can safely stop using web3 everywhere.